module-apps/application-phonebook/windows/PhonebookMainWindow.cpp

```diff
@@ -6,19 +6,21 @@
 #include "application-phonebook/data/PhonebookItemData.hpp"
 #include "application-phonebook/data/PhonebookStyle.hpp"
 
+#include <memory>
 #include <queries/phonebook/QueryContactGet.hpp>
 
 #include <header/AddElementAction.hpp>
 #include <header/SearchAction.hpp>
 #include <service-appmgr/Controller.hpp>
 #include <service-db/QueryMessage.hpp>
 #include <service-db/DBNotificationMessage.hpp>
+#include <utility>
```

Comment:
needed? or added by IDE?

Reply:
both were missing (and were added by some removed `more general include. I will double check includes though, thanks!